css/css-images/gradient/gradient-eval-004-ref.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <title>Gradient interpolation</title>
+  <link rel="author" title="Mike Bremford" href="mailto:mike@bfo.com">
+  <style>
+   .test {
+       width: 100px;
+       height: 100px;
+       background: rgb(0% 50% 50%);
```

Comment:
I'm afraid this is not the correct expected value, gradient interpolation is different from `color-mix()`. It's not equivalent to `color-mix( in srgb, rgb(0% 0% 50%), rgb(0% 100% 50%) )`.

@svgeesus WDYT?